src/components/AnalysisContent/ContentNavigation.js

```diff
@@ -55,13 +56,12 @@ AnalysisContentNavigation.propTypes = {
   labelTextStrong: PropTypes.string.isRequired,
   current: PropTypes.number.isRequired,
   content: PropTypes.shape({
-    body: PropTypes.arrayOf(
+    topics: PropTypes.arrayOf(
       PropTypes.shape({
-        id: PropTypes.string,
-        value: PropTypes.shape({
-          type: PropTypes.string,
-          title: PropTypes.string
-        })
+        ID: PropTypes.number,
```

Comment:
Any reason `ID` is in caps? And unless we're manipulating it as a number, it should probably just remain string.

Reply:
@kilemensi wordpress post object has caps `ID` and as number

Reply:
@kilemensi released we are not using `ID`, I will remove this from props